training/controllers/basic_funcionalities.py

```diff
@@ -20,6 +20,10 @@ def ver():
 @bp.route('/g')
 def test_g():
     if g.user is None:
-        return "No  hay usuario logueado", 411
+        return "No user logged", 401
     else:
-        return str(g.user), 200
+        user_dict = {}
+        for field in g.user.__dict__:
+            if field != '_sa_instance_state':
+                user_dict[field] = g.user.__dict__[field]
+        return user_dict, 200
```

Comment:
Maybe it's just better to go back to the previous version. This is not a good way of serializing the user data, besides the endpoint response format is not consistent on all cases.

Reply:
Okey, I will comment it and we will address it in the next PR.